examples/complete/main.tf

```diff
@@ -1 +1,14 @@
-#(empty)
+module "complete" {
```

Comment:
Hey @patrickmukumbu, just a slight change, I think the ou and account should be created separately in the main example so that we can show the account being created directly inside the ou..rather than adding it manually after creation...:)